### ISSUE TITLE
webpack css-loader failing on css validation

### ISSUE THREAD

**Issue by margielm** [angular/material#1215]
Hello,
In my project I am trying to use angular-material together with webpack. unfortunately for when I am trying to use css-loader to load angular-material.css it is loading on validation:

```
Module build failed: Error: Please check the validity of the CSS block starting from the line #6153
    at throwError (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:399:15)
    at checkStylesheet (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:1921:22)
    at Object.CSSPRules.stylesheet (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:365:40)
    at _getAST (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:409:38)
    at exports.srcToCSSP (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:2287:16)
    at Object.exports.srcToCSSP (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:2292:16)
    at Object.exports.parse (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/cssoapi.js:7:21)
    at Object.module.exports (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/index.js:18:18)
 @ ./src/scripts/entry.js 9:0-48

```

I have tried 0.6.1 and 0.7.0-rc3 and on both versions result was the similar.

**Comment by Frank3K** [comment]
Is the line number (6153) from the 0.7.0-rc3 or from the 0.6.1 file?

**Comment by margielm** [comment]
thats from 0.7.0-rc3, 
here is the stack trace from 0.6.1:
```
Module build failed: Error: Please check the validity of the CSS block starting from the line #5176
    at throwError (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:399:15)
    at checkStylesheet (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:1921:22)
    at Object.CSSPRules.stylesheet (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:365:40)
    at _getAST (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:409:38)
    at exports.srcToCSSP (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:2287:16)
    at Object.exports.srcToCSSP (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/gonzales.cssp.node.js:2292:16)
    at Object.exports.parse (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/node_modules/csso/lib/cssoapi.js:7:21)
    at Object.module.exports (/Users/margielm/workspace/jelatyna-gui/node_modules/css-loader/index.js:18:18)
 @ ./src/scripts/entry.js 9:0-48

```

**Comment by Frank3K** [comment]
The [start block starting at line 6153](https://github.com/angular/bower-material/blob/67213b37b137e2ec21577ddbb590291c7adb3777/angular-material.css#L6153-L6154) (0.7.0-rc3) does not contain anything odd as far as I can tell. The [W3C CSS Validator](http://jigsaw.w3.org/css-validator/validator) does report some issues, but not around those lines. Could it be that this is a webpack issue?

**Comment by mereskin** [comment]
Having the same issue with css file.

Fails with error "Error: Please check the validity of the CSS block starting from the line #2" when required as module: `require('my-module/foo.css')`.

File is valid css. When I copy the same file to project folder and require it as `require('./foo.css')` it works.

Loader is set up like that: `{ test: /\.css$/, loader: 'style-loader!css-loader' }`

**Comment by ThomasBurleson** [comment]
Fixed with 2f9eef70b5503b4fcb5c7cbe09712700403cc5a3.
@margielm, @mereskin - can you confirm with lastest from [/bower-material/angular-material.css](https://github.com/angular/bower-material/blob/master/angular-material.css)

**Comment by margielm** [comment]
the issue is still there. ust the line number has changed. 
```
ERROR in ./~/css-loader!./~/exports-loader?angular!./bower_components/angular-material/angular-material.css
Module build failed: Error: Please check the validity of the CSS block starting from the line #6718
```

As a workaround I have used raw loader.
